app-ios/tutanota/Sources/Crypto/Argon2idFacade.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import argon2
+
+class Argon2idFacade {
```

Comment:
don't know if it should be called "facade" because it's not one of the IPC facades.
Also you don't have to wrap it in a class probably?

Reply:
OK 6ffde0e53602a618fc5f78ebaa485d44616f0a62